agent/wpthook/results.cc

```diff
@@ -483,15 +483,36 @@ void Results::SavePageData(OptimizationChecks& checks){
     CStringA result;
     CStringA buff;
 
+    // The browser agent sends metrics in milliseconds since UNIX EPOC. We need to convert the start time to
+    // a UNIX timestamp in MS and account for the offset applied to the requests (_start)
+
+    // we need a UNIX EPOC of start time in ms. So,
+    // 1. lets convert the FILETIME to a 64 LONGLONG
+    LONGLONG ll_start = (LONGLONG)_test_state._start_time.dwLowDateTime + ((LONGLONG)(_test_state._start_time.dwHighDateTime) << 32LL);

```

Comment:
The compiler refuses.

I think my implementation is the correct way of doing a cast to LONG LONG.
https://support.microsoft.com/en-us/kb/188768